burgershop-finder/LocationPermissionViewModel.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  LocationPermissionViewModel.swift
+//  burgershop-finder
+//
+//  Created by e.vanags on 01/10/2019.
+//  Copyright © 2019 esesmuedgars. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import RxCocoa
+
+final class LocationPermissionViewModel {
+
+    private let locationService: LocationServiceProtocol
+
+    let descriptionText = "We'll help you find burger venues in Tallin. Grant permission to have your location displayed."
+
+    private let _buttonTitle = BehaviorSubject<String>(value: "")
```

Comment:
I guess this was made as `BehaviorSubject<String>` since first value is emitted on `init()`, and UI elements subscribe afterwards, which results in loss of first value. Since default value is not used, could replace with `ReplaySubejct<String` with buffer size of one.
```suggestion
    private let _buttonTitle = ReplaySubejct<String>.create(bufferSize: 1)
```